### ISSUE TITLE
Hooks don't work after standard install and configuration

### ISSUE THREAD

**Issue by michaelterryio** [phenixdigital/phoenix_storybook#284]
The reference fly.io deploy emits the same warnings/errors that I'm seeing in my app.

 See phenixdigital/phoenix_storybook_sample#4

**Comment by michaelterryio** [comment]
Embedding stories in an iframe avoids the problem, so for now we can follow the guide here:

https://hexdocs.pm/phoenix_storybook/sandboxing.html#4-enabling-iframe-rendering

...e.g.:

```patch
defmodule MyAppWeb.Storybook.AlpineComponent do
  use PhoenixStorybook.Story, :live_component

  def component, do: MyAppWeb.Components.Alpine
  def description, do: "Generic Alpine component"
  def attributes, do: [
    %Attr{id: :id, type: :string},
    %Attr{id: :name, type: :string, required: true},
    %Attr{id: :data, type: :map, required: true}
  ]
  def slots, do: []
+  def container do
+    :iframe
+  end
  def variations, do: [
    %Variation{
      id: :default,
      attributes: %{
        name: "foobox",
        data: %{foo: "bar"}
      }
    }
  ]
end
```

**Comment by danturn** [comment]
Im having the same issue in our app with Hooks :(

**Comment by justlampin** [comment]
@michaelterryio @danturn it looks like this was fixed in `v0.5.3`, which hasn't been released yet: https://github.com/phenixdigital/phoenix_storybook/commit/daf7bfce79d201e44cd4e5575289ecc10ba616c7

you could get the fix now by pulling your dependency from github, e.g. in `mix.exs`:
`{:phoenix_storybook, github: "phenixdigital/phoenix_storybook"}`

in which case you'll need to run `mix dev.storybook` to build the static assets (the release from `hex.pm` has them prebuilt, the github dependency does not)

**Comment by michaelterryio** [comment]
Ya know, I actually saw that issue, I think coming off Google, and it didn't occur to me that it was prerelease. Thanks.
